src/legacy/ui/public/utils/collection.ts

```diff
@@ -33,10 +33,10 @@ import _ from 'lodash';
  * @return {array} - the objs argument
  */
 export function move(
-  objs: object[],
+  objs: any[],
```

Comment:
Why did you broaden the type here?

Reply:
Please see tests, looks like that function works fine with primitive types too

![image](https://user-images.githubusercontent.com/20072247/71961023-7e19c780-3207-11ea-905e-860712968052.png)